scripts/lib.star

```diff
@@ -145,6 +145,12 @@ def master_pipelines(edition):
             name='publish-master', edition=edition, trigger=trigger, steps=publish_steps,
             depends_on=['build-master', 'windows-master',], install_deps=False, version_mode=version_mode,
         ))
+
+        notifyTrigger = dict(trigger, status = ['failure'])
```

Comment:
Can you write in snake case instead of camel case, since it's standard in Starlark?

Reply:
Sure will fix. I thought you could do `trigger + { 'status': ['failure']` similar to jsonnet, but it didn't work so used dict instead. Do you know?

Reply:
@marefr `dict` is probably fine. There's also a dict literal merging syntax in Python, but not sure if Starlark supports it.